app/tests/unit/layout/default-layout.test.tsx

```diff
@@ -3,8 +3,29 @@ import {shallow} from 'enzyme';
 import {DefaultLayoutComponent} from 'layouts/default-layout';
 
 describe('The DefaultLayout component', () => {
-  it('matches the last accepted Snapshot', () => {
+  it('when logged out matches the last accepted Snapshot', () => {
     const wrapper = shallow(<DefaultLayoutComponent session={null} />);
     expect(wrapper).toMatchSnapshot();
   });
+
+  it('when logged in matches the last accepted Snapshot', () => {
+    const mockRefType: any = null;
+    const user = {
+      __typename: 'qq',
+      firstName: 'Test',
+      lastName: 'Tester',
+      emailAddress: 'test-tester@testing.com',
+      ' $fragmentRefs': mockRefType
```

Comment:
```suggestion
      ' $fragmentRefs': {
        UserProfileDropdown_user: true
      }
```
This is just listing the fragments that are in the query. Its only purpose when unit testing is to make typescript happy (same as `' $refType'`)

Reply:
Type '{ firstName: string; lastName: string; emailAddress: string; ' $fragmentRefs': { UserProfileDropdown_user: boolean; }; }' is not assignable to type '{ readonly " $fragmentRefs": FragmentRefs<"UserProfileDropdown_user">; }'.
  The types of '" $fragmentRefs".UserProfileDropdown_user' are incompatible between these types.
    Type 'boolean' is not assignable to type 'true'.

I was getting this before, not sure why it's complaining for this component since I saw others similar to this.

Reply:
just having to cast everything with `as` in the end for these tests, seems like other areas of the codebase are doing that as well.